server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileReset.java

```diff
@@ -54,19 +55,22 @@
   private final QProfileFactory factory;
   private final RuleActivator activator;
   private final ActiveRuleIndexer activeRuleIndexer;
+  private final DefaultOrganizationProvider defaultOrganizationProvider;
   private final ProfileDefinition[] definitions;
 
-  public QProfileReset(DbClient db, RuleActivator activator, ActiveRuleIndexer activeRuleIndexer, QProfileFactory factory,
-    ProfileDefinition[] definitions) {
+  public QProfileReset(DbClient db, RuleActivator activator, ActiveRuleIndexer activeRuleIndexer, QProfileFactory factory, DefaultOrganizationProvider defaultOrganizationProvider,
```

Comment:
I also thought so, but I got pico-errors when dropping the other constructor. I'll try again...